docs/application/native/guides/multimedia/webrtc.md

```diff
@@ -0,0 +1,447 @@
+    }
+
+    void _ice_gathering_state_change_cb(webrtc_h webrtc, webrtc_ice_gathering_state_e state, void *user_data)
+    {
+        /* When setting local description, it'll be changed to WEBRTC_ICE_GATHERING_STATE_GATHERING. If the gathering work is done, it'll be changed to WEBRTC_ICE_GATHERING_STATE_COMPLETE. */
+    }
+
+    void _ice_connection_state_change_cb(webrtc_h webrtc, webrtc_ice_connection_state_e state, void *user_data)
+    {
+        /* After setting both description and ICE candidates from remote peer, it'll be changed from WEBRTC_ICE_CONNECTION_STATE_CHECKING to WEBRTC_ICE_CONNECTION_STATE_COMPLETED. */
+    }
+
+    void webrtc_func(void)
+    {
+        int ret;
+        webrtc_h webrtc;
+        ...
+        ret = webrtc_set_peer_connection_state_change_cb(webrtc, _peer_connection_state_change_cb, user_data);
+        ret = webrtc_set_signaling_state_change_cb(webrtc, _signaling_state_change_cb, user_data);
+        ret = webrtc_set_ice_gathering_state_change_cb(webrtc, _ice_gathering_state_change_cb, user_data);
+        ret = webrtc_set_ice_connection_state_change_cb(webrtc, _ice_connection_state_change_cb, user_data);
+        ...
+        ret = webrtc_start(webrtc);
+        /* Do negotiation */
+    }
+    ```
+
+<a name="media_render"></a>
+## Rendering Audio/Video Data
+
+You can decide how to handle audio/video streaming data received from remote peer by using functions provided in this API set. You can also render sending audio/video data on local target device.
+
+1. To get notified of creation of audio or video track from remote peer, use `webrtc_set_track_added_cb()` function:
+
+    ```c
+    void _track_added_cb(webrtc_h webrtc, webrtc_media_type_e type, unsigned int track_id, void *user_data)
+    {
+        int ret;
+        some_app_data_s *data = (some_app_data_s *)user_data;
+
+        if (type == WEBRTC_MEDIA_TYPE_AUDIO) {
+            ret = sound_manager_create_stream_information(SOUND_STREAM_TYPE_MEDIA, NULL, NULL, &data->stream_info);
+            ret = webrtc_set_sound_stream_info(webrtc, track_id, data->stream_info);
+
+        } else if (type == WEBRTC_MEDIA_TYPE_VIDEO) {
+            /* To render video data to video overlay, use window id */
+            ret = webrtc_set_display(webrtc, id, WEBRTC_DISPLAY_TYPE_OVERLAY, data->win_id);
+            ... or ...
+            /* To render video data to EVAS image object */
+            ret = webrtc_set_display(webrtc, id, WEBRTC_DISPLAY_TYPE_EVAS, data->evas_image_object);
+        }
+    }
+
+    void webrtc_func(void)
+    {
+        int ret;
+        webrtc_h webrtc;
+        ...
+        ret = webrtc_set_track_added_cb(webrtc, _track_added_cb, data);
+        ...
+        ret = webrtc_start(webrtc);
+        ...
+        /* After finishing negotiation, _track_added_cb() could be called if receiving audio/video data from remote peer exists */
+    }
+    ```
+    > **Note**
```

Comment:
For alerts, please use alert formatting.
Ref: https://github.com/Samsung/tizen-docs/blob/master/styleguide/custom-style.md#alerts-note-tip-important-caution-warning
```suggestion
    > [!NOTE]
```